src/RequestBuilder/RequestBuilderFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Lmc\Matej\Http\RequestManager;
+
+/**
+ * Factory to create concrete RequestBuilder which helps you to create request for each Matej API
+ */
+class RequestBuilderFactory
+{
+    /** @var RequestManager */
+    private $requestManager;
+
+    public function __construct(RequestManager $requestManager)
+    {
+        $this->requestManager = $requestManager;
+    }
+
+    public function itemPropertiesSetup(): ItemPropertiesSetupRequestBuilder
```

Comment:
`itemPropertiesSetupBuilder`?

Reply:
Its already inside a BuilderFactory, so this would be imho not necessary "noise" in the method naming...

Also we're trying to make the builder API as much readable as possible, so not naming the class names in a full name (but using "natural language" aliases) is kind of the goal. 

Does it make sense :-)?